Customizing the deployment changes (#88)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_creating-a-servicetelemetry-object-in-openshift.adoc

// Module included in the following assemblies:
//
// <List assemblies here, each on a new line>

// This module can be included from assemblies using the following include statement:
// include::<path>/proc_creating-a-servicetelemetry-object-in-openshift.adoc[leveloffset=+1]

// The file name and the ID are based on the module title. For example:
// * file name: proc_doing-procedure-a.adoc
// * ID: [id='proc_doing-procedure-a_{context}']
// * Title: = Doing procedure A
//
// The ID is used as an anchor for linking to the module. Avoid changing
// it after the module has been published to ensure existing links are not
// broken.
//
// The `context` attribute enables module reuse. Every module's ID includes
// {context}, which ensures that the module has a unique ID even if it is
// reused multiple times in a guide.
//
// Start the title with a verb, such as Creating or Create. See also
// _Wording of headings_ in _The IBM Style Guide_.

> [id="creating-a-servicetelemetry-object-in-openshift"]

Author: Separate file for this module so that we can x-ref from a module in a separate assembly

== Creating a ServiceTelemetry object in {OpenShiftShort}

> To deploy the Service Telemetry Framework, you must create an instance of `ServiceTelemetry` in {OpenShiftShort}. By default, `eventsEnabled` is set to false. If you do not want to store events in ElasticSearch, ensure that `eventsEnabled` is set to false. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.

Reviewer: ```suggestion
To deploy the {Project}, you must create an instance of `ServiceTelemetry` in {OpenShiftShort}. By default, `eventsEnabled` is set to false. If you do not want to store events in ElasticSearch, ensure that `eventsEnabled` is set to false. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.
```

The following core parameters for a `ServiceTelemetry` manifest are available:

|===
| Parameter | Description | Default Value

| `eventsEnabled` | Enable events support in {ProjectShort}. Requires prerequisite steps to ensure ElasticSearch can be started. For more information, see <<subscribing-to-elastic-cloud-on-kubernetes-operator>>. | `false`

| `metricsEnabled` | Enable metrics support in {ProjectShort}.| `true`

| `highAvailabilityEnabled` | Enable high availability in {ProjectShort}. For more information, see <<high-availability_advanced-features>>. | `false`

| `storageEphemeralEnabled` | Enable ephemeral storage support in {ProjectShort}. For more information, see <<ephemeral-storage_advanced-features>>. | `false`
|===

[discrete]
=== Procedure

. To store events in ElasticSearch, set `eventsEnabled` to true during deployment:

> 

Reviewer: ```suggestion
```

+
[source,bash]
----
oc apply -f - <<EOF
apiVersion: infra.watch/v1alpha1
kind: ServiceTelemetry
metadata:
  name: stf-default
  namespace: service-telemetry
spec:
  eventsEnabled: true
  metricsEnabled: true
EOF
----

. To view the {ProjectShort} deployment logs in the Service Telemetry Operator, run the `oc logs` command:

> 

Reviewer: ```suggestion
```

+
----
oc logs $(oc get pod --selector='name=service-telemetry-operator' -oname) -c ansible
----

> . View the pods and the status of the pods to determine that all workloads are operating nominally:

Author: Line 71 and 79 still have the same text. I think I can remove line 71 and then the PLAY RECAP output becomes the output of the oc logs command on line 68. Is that correct?

Author: @leifmadsen Can I get your opinion on this please and thanks

Reviewer: Yes that looks right
